frontend/fingertips-frontend/components/forms/IndicatorSearchForm/indicatorSearchActions.test.ts

```diff
@@ -64,7 +78,10 @@ describe('Search actions', () => {
   });
 
   it('should redirect to search results if indicator and areas are provided', async () => {
-    const formData = getMockFormData({ indicator: 'boom' });
+    const formData = getMockFormData({
+      indicator: 'boom',
+      searchState: areasSelectedState,
+    });
 
     await searchIndicator(initialStateWithAreas, formData);
```

Comment:
Should this use `initialStateWithoutAreas` too, as the test above does?
```suggestion
    await searchIndicator(initialStateWithoutAreas, formData);
```

Reply:
I think this is right. Since this one is testing `should redirect to search results if indicator and areas are provide`

The one above however is wrong and that does need changing to `initialStateWithAreas` since that is testing `should redirect to search results if indicator and areas are provided`.

Reply:
Updated.